apps/native/app/src/stores/offline-store.ts

```diff
@@ -0,0 +1,52 @@
+import createUse from 'zustand'
+import create from 'zustand/vanilla'
+import { isDefined } from '../utils/is-defined'
+
+interface State {
+  pastIsConnected: boolean
+  isConnected: boolean | null
+  bannerVisible: boolean
+  bannerHasBeenShown: boolean
+}
+
+interface Actions {
```

Comment:
~They also some have the actions declared as an object inside the state, like` environment-store `for example instead of as a separate. Do we want to keep that consistency or rather have all props and actions on the same level like here?

Reply:
thats a good point. I will update